go-kit/metrics/provider/librato/librato.go

```diff
@@ -211,35 +225,119 @@ func prefixName(prefix, name string) string {
 	return prefix + "." + name
 }
 
+// keyName is used as the map key for counters, gauges, and histograms
+// and incorporates the name and the labelValues.
+func keyName(name string, labelValues ...string) string {
+	if len(labelValues) == 0 {
+		return name
+	}
+
+	l := len(labelValues)
+	parts := make([]string, 0, l/2)
+	for i := 0; i < l; i += 2 {
+		parts = append(parts, labelValues[i]+":"+labelValues[i+1])
+	}
+	return name + "." + strings.Join(parts, ".")
+}
+
+// metricName returns the name we'll use for the metric depending on
+// whether we're using tags or not.
+//
+// GIVEN
```

Comment:
looks like this behavior is captured in the `TestProviderMetricName` test so i think this comment block is unnecessary.